lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,38 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden" tabindex="-1">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Your team could not created!") %></h3>
```

Comment:
Typo: Change to `"Your team could not be created!"` (correct me if I'm wrong, @catong).